src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,21 @@
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const NavBar = () => {
+  const getIsActive = ({ isActive }: { isActive: boolean }) =>
+    cn('navbar-item', {
+      'has-background-grey-lighter': isActive,
+    });
```

Comment:
I think it will be better to move this function outside our component. 

Currently, we create a new function on each `NavBar` re-render.